### PR TITLE
[SDK-1097][Docs] Update SPM download instructions to use new `braze-ios-sdk` repo

### PR DESCRIPTION
**Pull request by hokstuff**
# Pull Request/Issue Resolution

**Description of Change:**
We have added a new repository for apps integrating through Swift Package Manager. The repository is live right now (https://github.com/braze-inc/braze-ios-sdk), but we may do small tweaks before officially adding it to our public docs. We will align the docs release with our upcoming iOS release (likely called 3.33.2)

The previous screenshots for the `appboy-ios-sdk` now display `braze-ios-sdk`.

https://jira.braze.com/browse/SDK-1097

FYI @mattlsteinberg @malandr2 

### Is this change associated with a Braze feature/product release?
- [x] Yes (__3.33.2, sometime in the first week of March__)
- [ ] No

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [x] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [x] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md [hunk 1]
```diff
@@ -36,6 +38,9 @@ Select the package that best fits your needs and click _Finish_:
   - Includes `AppboyKit` automatically.
 - `AppboyKit`
   - Best suited if you don't need to use any of the UI components provided by Braze (e.g. Content Cards, In-App Messages, etc.).
+- `AppboyPushStory`
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md:41]: Note version availability

**Review by Bucimis — DISMISSED** [review]
good stuff!

File: _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md (hunk 1 above)
Comment (hokstuff) [reply _docs/_developer_guide/platform_integration_guides/ios/initial_sdk_setup/swift_package_manager.md:40]: Moved the line above mentioning the version to this section

**Review by lowip — APPROVED** [review]
lgtm

**Review by chshapiro — APPROVED** [review]

**Review by KellieHawks — APPROVED** [review]
looks good to me! @hokstuff, please let me know when you would like this merged this week. Happy to merge today.

**Comment by hokstuff** [comment]
@KellieHawks We're planning to merge this in around the time of the next release - I'll keep you updated once we know when that is, but it'll be one of the next few business days!

**Comment by hokstuff** [comment]
@KellieHawks - We just released version 3.34.0 so this is ready to be deployed when you are :)

**Pull request merged**